pages/index.html

```diff
@@ -127,10 +127,25 @@ <h2 class="bannerbox__title">I need a ride</h2>
                                 <input type="checkbox" name="TwoWayTripNeeded" id="rideReturn" /> I need a two-way trip.
                             </label>
                         </div>
+                        <div class="form-group checkbox">
+                            <label for="riderDoesNotSpeakEnglish">
+                                <input type="checkbox" name="TwoWayTripNeeded" id="riderDoesNotSpeakEnglish" /> I can only communicate in a language other than English.
```

Comment:
`name="TwoWayTripNeeded"` is incorrect, and duplicates the previous checkbox. Please change the name attribute to a unique value. Maybe just `name="riderDoesNotSpeakEnglish"`?